src/components/List.js

```diff
@@ -1,12 +1,54 @@
 import React from 'react';
 import useFirestore from '../hooks/useFirestore';
+import { db } from '../lib/firebase';
 
 const List = ({ token }) => {
   const { docs } = useFirestore(token);
+
+  const checkPurchasedDate = (purchasedDate) => {
+    if (purchasedDate === null) {
+      return false;
+    }
+    const today = new Date();
+    let daySincePurchased = purchasedDate.toDate();
+
+    daySincePurchased.setDate(daySincePurchased.getDate() + 1); // Add one day to the last purchased date
+
+    // If today > daySincePurchased at least 24 hours have passed, return false to uncheck box
+    if (
+      Date.parse(today.toString()) >= Date.parse(daySincePurchased.toString())
```

Comment:
`today` and `daySincePurchased` are both of type `Date` right? Is it possible to remove the `.toString` and the `Date.parse` and just use them directly as Date objects? (I could be wrong, too! I'm not sure)

Reply:
Looking into this now. I think we had that in to console log something at some point. I've deleted and it still works. Going to do some more testing before I push the change 😄